api/websocket/messagebuffer/messagebuffer_test.go

```diff
@@ -0,0 +1,147 @@
+package messagebuffer
+
+import (
+	"encoding/hex"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/nknorg/nkn/v2/pb"
+	"github.com/stretchr/testify/require"
+)
+
+var clientId [][]byte = [][]byte{[]byte{byte(0x01)}, []byte{byte(0x02)}, []byte{byte(0x03)}, []byte{byte(0x04)}}
+
+var msg *pb.Relay = &pb.Relay{SrcIdentifier: "from", DestId: []byte("to"), Payload: []byte("payload"),
+	MaxHoldingSeconds: 10}
+
+var msgBuffer *MessageBuffer
+
+// to create random MaxHoldingSeconds
+func addMsg(clientId []byte, m *pb.Relay) {
+	if msgBuffer == nil {
+		msgBuffer = NewMessageBuffer()
+	}
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Random `MaxHoldingSeconds` is NOT enough to test all the problems we have encountered. We need to add a set of new data each time need to test a new problem.

Reply:
yes. I rewroted all test cases for specific unit test target.